### DISCUSSION TITLE
Auto RefreshThumbnails

### DISCUSSION THREAD

**Discussion by tazmologist** [fastaddons/GroupSpeedDial#335] [Q&A] [answered]
I'm new to this product and I may have missed it, but is there a way to have the thumbnails update from the target site automatically?

**Comment by Juraj-Masiar** [comment, marked as answer]
Yes and no :). It's a bit complicated.
When you right click a dial / Edit dial, and then click the "Auto-reload" tab, you can setup automatic thumbnail refresh.
However, currently only Firefox can create thumbnails "in the background" without bothering you at all.
Chrome and other Chromium based browsers (Edge) will need to create new window where it creates the thumbnail, and it cannot be hidden.

There is plan to refresh the thumbnail when you visit the page:
https://github.com/fastaddons/GroupSpeedDial/issues/290
Hopefully I will get to this soon.

And there is also plan for "Cloud reload" service, which will create thumbnails in the cloud. I was implementing this a while ago, but I had to pause it for a moment since it got a bit too complex and I realized I need to finish some other features first.

**Reply by tazmologist** [reply]
That was MOST helpful.  I can deal with the popup window in Chrome....but this may be the "thing" that pushes me over the line to using Edge full time.  :) Thanks for the fast response!!

**Reply by MaJorDan383** [reply]
So even with tabs being able to open in the background, this is impossible?

**Reply by Juraj-Masiar** [reply]
> So even with tabs being able to open in the background, this is impossible?

Yes, it's not possible, there are two big issues:
- Chrome won't fully load page in a "not active" tab (to save CPU), this is partially affecting also Firefox
- Chrome doesn't allow taking screenshot of a "not active" tab (unlike Firefox), so the tab have to be focused in order to screenshot it
